EdFi.Buzz.Database/eng/Windows/install.ps1

```diff
@@ -69,6 +72,22 @@ function Install-Migrations {
     }
 }
 
+function Install-SampleData {
+    if ($LoadSampleData) {
+        try {
+            Push-Location -Path $distFolder
+            Write-Host "Installing sample data" -ForegroundColor Magenta
+            &db-migrate up:sample-data --silent "$DbName" --config ./migrate-database.json | Out-File -FilePath $logFile -Append
+            Write-Host "Sample data installed" -ForegroundColor Magenta
+            Pop-Location
+        }
+        catch {
+            Write-Error "Sample data was not installed"
+            throw
```

Comment:
A couple of things I learned from FIF-299.

Favor `$_` in the catch block for handling exceptions. You can Write-Host with the ScriptStackTrace to produce the error that was being swallowed when I ran.

Also, do use Write-Host over Write-Error. In Octopus Deploy, the ErrorActionPreference is Stop. That means the first Write-Error will cause Octopus to stop execution of the PowerShell script. Then, we can throw `$_` just like the other catch blocks to report the exception up the trace.